force-app/main/default/lwc/attendance/attendance.html

```diff
@@ -0,0 +1,34 @@
+<template>
+    <lightning-card>
+        <div slot="title">
+            <lightning-icon icon-name="custom:custom15" size="small"></lightning-icon>
+            <span class="slds-p-left_x-small">
+                {header}
+            </span>
+        </div>
+
+        <div class="slds-border_top">
+            <div class="slds-scrollable" style="max-height: 500px;">
+                <template if:true={serviceDeliveries}>
+                    <template for:each={serviceDeliveries} for:item="delivery">
+                        <c-attendance-row
+                            record={delivery}
+                            field-set={fieldSet}
+                            unit-of-measurement={unitOfMeasurement}
+                            key={delivery.index}
+                        ></c-attendance-row>
+                    </template>
+                </template>
+            </div>
+        </div>
+
+        <div slot="footer" class="slds-text-align_right">
```

Comment:
So when there are no participant records and we look at the attendence component in the UI, we see a blank screen with the Submit button. I am assuming that we should not be seeing the submit button. Would that be addressed in the following stories